docs/development/contributing-to-docs.md

```diff
@@ -0,0 +1,139 @@
+  5. Docusaurus?
+
+## Contributing
+
+  1. On branch ```master```, fetch the latest commit:
+
+  ~~~~
+  git fetch upstream
+
+  remote: Enumerating objects: 397, done.
+  remote: Counting objects: 100% (341/341), done.
+  remote: Compressing objects: 100% (181/181), done.
+  remote: Total 397 (delta 118), reused 255 (delta 98), pack-reused 56
+  Receiving objects: 100% (397/397), 266.19 KiB | 16.64 MiB/s, done.
+  Resolving deltas: 100% (118/118), completed with 44 local objects.
+  From https://github.com/apache/druid
+     819d706082..6c9f926e3e  0.21.0       -> upstream/0.21.0
+     8a3be6bccc..84aac4832d  master       -> upstream/master
+   * [new tag]               druid-0.21.0 -> druid-0.21.0
+
+  ➜ git reset --hard upstream/master
+  HEAD is now at 84aac4832d Add feature to automatically remove rules based on retention period (#11164)
+  ~~~~
+
+  Now you're up to date.
+
+  2. Create your working branch:
+  ~~~~
+  git checkout -b my-work
+  ~~~~
+  3. Make your changes, add, and commit:
+  ~~~~
+  git add my-change.md
+  git commit -m "i made some changes"
+  ~~~~
+  4. Test changes locally.
+  5. Push your changes to your fork: ```origin```
+  ~~~~
+  git push --set-upstream origin my-work
+  ~~~~
+  6. Go to GitHub for the Druid repo. It should realize you have a new branch in your fork. Create a pull request from your for ```my-work``` to ```master``` in the Druid repo.
+
+## Style guide
+
+Before publishing new content or updating an existing topic, audit your documentation using this checklist to make sure your contributions align with existing documentation.
+
+* Use descriptive link text. If a link downloads a file, make sure to indicate this action
+* Use present tense where possible
+* Avoid negative constructions when possible
+* Use clear and direct language
+* Use descriptive headings and titles
+* Avoid using a present participle or gerund as the first word in a heading or title
+* Use sentence case in document titles and headings
+* Don’t use images of text or code samples
+* Use SVG over PNG if available
+* Provide an equivalent text explanation with each image
+* Use the appropriate text-formatting
+* Put conditional clauses before instructions
+* Avoid gender-specific pronouns, instead use they
+* Use second person — you instead of we
+* When American spelling is different from Commonwealth/"British" spelling, use the American spelling
+* Don’t use terms considered disrespectful. Refer to the Google’s [Word list](https://developers.google.com/style/word-list) for guidance and alternatives
+* Use straight quotation marks and straight apostrophes
+* Introduce a list, a table, or a procedure with an introductory sentence
```

Comment:
In Markdown files, wrap lines at 80 characters (some like 120 in code.) No need to have one paragraph per line, except in tables.

Reply:
@paul-rogers , can we not introduce this style now. None of the current docs follow it and I worry that it could cause confusion. For example, if someone were to edit an existing doc and use a line-breaking tool to fix their markdown, then it would possibly make the pr affect a lot more lines than they intend.